utils/lib/rr_graph/channel.py

```diff
@@ -629,12 +590,14 @@ def create_track(self, t, idx=None):
 
         # Find start and end
         # TODO: BI should maybe sort
```

Comment:
this TODO has been fixed per the min/max below

Reply:
Done.